packages/web-components/src/radio-group/radio-group.ts

```diff
@@ -0,0 +1,38 @@
+import { colorNeutralForegroundDisabled } from '@fluentui/web-components';
+import { attr } from '@microsoft/fast-element';
+import { FASTRadioGroup } from '@microsoft/fast-foundation';
+
+/**
+ * The base class used for constructing a fluent-radio-group custom element
+ * @public
+ */
+export class RadioGroup extends FASTRadioGroup {
+  /**
+   * sets radio layout styles
+   *
+   * @public
+   * @remarks
+   * HTML Attribute: stacked
+   */
+  @attr({ mode: 'boolean' })
+  public stacked: boolean = false;
+
+  protected createLocalTokens(): void {
+    if (!this.$fastController.isConnected) {
+      return;
+    }
+
+    if (this.disabled) {
```

Comment:
why not juts assign these all in css using `:host([disabled])`? Then we can avoid extra javascript

Reply:
These CSS variables are for styling elements that are slotted within `Radio` when the `RadioGroup` is `disabled`

Reply:
But css variables work across shadowRoots. These could be defined in the styles of `radio-group` and be used by `radio` and `label`

Reply:
ahh I see. That's much better. Thank you!